pkgs/top-level/all-packages.nix

```diff
@@ -8181,14 +8180,6 @@ in
     stripped = false;
   }));
 
-  libstdcxxHook = makeSetupHook
```

Comment:
We may need a migration or warning for this.

It may not be actively used though

Reply:
aliases.nix should be enough.

Reply:
I am not sure what should be aliased?

Reply:
Not aliased but we need a `throw` statement in `aliases.nix` that generates a warning.

Reply:
Ah! sounds good.